src/components/workspace/settings/BillingOverview.vue

```diff
@@ -60,72 +94,182 @@
             class="billing-card__volume-progress"
           />
         </div>
-      </div>
+      </section>
     </div>
     <div class="billing-card__buttons">
-      <button
-        class="button button--submit billing-card__button"
-        @click="processPayment(100)"
-      >
-        {{ $t('billing.pay') }} 100$
-      </button>
-      <button
-        class="button button--submit billing-card__button"
-        @click="processPayment(1000)"
-      >
-        {{ $t('billing.pay') }} 1000$
-      </button>
-      <button
-        class="button button--submit billing-card__button"
-        @click="processPayment()"
-      >
-        {{ $t('billing.payCustomAmount') }}
-      </button>
-      <button class="button button--submit billing-card__button billing-card__button--invoice">
-        <Icon
-          class="billing-card__invoice-icon"
-          symbol="invoice"
-        />
-        {{ $t('billing.requestInvoice') }}
-      </button>
+      <UiButton
+        v-for="(button, index) in buttons"
+        :key="'button:' + index"
+        :submit="button.style === 'primary'"
+        :content="button.label"
+        @click="button.onClick"
+      />
+    </div>
+    <div
+      v-if="isAutoPayOn"
+      class="billing-card__autopay-is-on"
+    >
+      {{ 'Autopay is on.The next payment date:' }} {{ subExpiredDate | prettyDateFromDateTimeString }}
```

Comment:
```suggestion
      {{ 'Autopay is on. The next payment date:' }} {{ subExpiredDate | prettyDateFromDateTimeString }}
```
тут не пропущено `$t`?